doc/widgets/univariatepolynomialregression.rst

```diff
@@ -40,40 +41,45 @@ Regressor coefficients if it has them.
 Description
 -----------
 
-This widget interactively shows regression line using different regressors from Orange3 Regression module provided on
-the input. In widget **Polynomial expansion** can be set. Polynomial expansion is regulation of the degree
-of polynom that is used to transform input data and has an effect on the shape of a curve. If polynomial expansion is
-set to 1 it means that untransformed data are used in regression.
+This widget interactively shows regression line using any of the regressors from *Orange3 Regression* module. 
+In the widget, `polynomial expansion <https://en.wikipedia.org/wiki/Polynomial_expansion>`__ can be set. 
+Polynomial expansion is a regulation of the degree of polynom that is used to transform the input data and has an effect 
+on the shape of a curve. If polynomial expansion is set to 1 it means that untransformed data are used in the regression.
 
-.. figure:: images/polyregressionmain.png
+.. figure:: images/polynomial-regression-stamped.png
 
-1. Provide data with at least two continuous attributes on input. Also provide any regression learner on input (if
-   not provided *Linear Regression* is used.
+1. Regressor name. Input data needs least two continuous attributes.
```

Comment:
I was debating this myself. The main idea is to keep the signal part quite short, as explanation should come in the 'Description' section. Still not sure what is a better option, but after some discussion in our lab, I would prefer to keep it in the 'Description' part if you agree. :)